Makefile

```diff
@@ -400,10 +369,8 @@ changelog:
 .PHONY: install-tools
 install-tools:
 	go install github.com/wadey/gocovmerge
-	go install golang.org/x/lint/golint
 	go install github.com/mjibson/esc
-	go install github.com/securego/gosec/cmd/gosec
-	go install honnef.co/go/tools/cmd/staticcheck
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.42.0
```

Comment:
Add this to the tools so CI is able to run golangci-lint, and ensure devs also have the right tools installed to test linting.